tests/resources/test_predictor.py

```diff
@@ -50,27 +51,80 @@ def test_delete():
 def test_archive_and_restore(valid_label_fractions_predictor_data):
     session = mock.Mock()
     pc = PredictorCollection(uuid.uuid4(), session)
-    session.get_resource.return_value = valid_label_fractions_predictor_data
+    entity = deepcopy(valid_label_fractions_predictor_data)
+    entity["metadata"]["archived"] = entity["metadata"]["created"]
+    session.put_resource.return_value = entity
```

Comment:
Right, but this, again, seems like a perfect Factory candidate.

Reply:
Oh, your wording and the line you tagged were unclear. You're talking about the `deepcopy`. Because it's data for a predictor with a specific instance type, so not a great fit for a factory.